policy/Makefile

```diff
@@ -65,6 +65,10 @@ ifeq (x$(DISTRO),$(filter x$(DISTRO),xRHEL4 xRHEL5 xRHEL6))
 TARGETS:=$(filter-out test_overlayfs.te test_mqueue.te, $(TARGETS))
 endif
 
+ifeq (x$(DISTRO),$(filter x$(DISTRO), xRHEL6))
+TARGETS:=$(filter-out test_ibpkey.te, $(TARGETS))
+endif
+
```

Comment:
I'd just add test_ibpkey.te to line 65 and exclude it on RHEL4, RHEL5, and RHEL6 (I assume this doesn't work on any of them).

Reply:
We do not officially support RHEL 4 anymore so we have not run the suite on that system. However, the test passes on RHEL 5 and doesn't complain about any syntax errors so we don't want to exclude it there.

Reply:
i was confused because the dev_rw_infiniband_dev interface isn't defined on RHEL5 either, but it doesn't matter because we long ago forked the RHEL5 (and RHEL4) policies under policy/redhat/{5,4}.  So I guess your change is fine.